docs/resources/guides/coding-a-page.mdx

```diff
@@ -20,22 +25,870 @@ You will finish this tutorial with a strong opinion from us on how to best devel
 - Page Layout
 - UI Component Organization
 - Component Variants
+  <br />
+  <br />
 
 ## Page Layout
 
 We are going to have two layouts one for smaller screens and one for larger screens. Luckily, today we have CSS Grid & Flex Box. We will be using these to create our layouts. On small screens, the recipe when opened will display on top of the search results. On larger screens, the recipe will be displayed to right of the search results, the search results will be displayed to the left of the recipe in 4 columns.
 
+### Container
+
+```jsx
+import React from "react";
+import { Container, styled } from "@washingtonpost/wpds-ui-kit";
+
+const StyledContainer = styled(Container, {
+  backgroundColor: "$blue600",
+  minHeight: "100vh",
+});
+
+export default function Home() {
+  return <StyledContainer>Container</StyledContainer>;
+}
+```
+
+WPDS ships with a responsive Container component that centers its content and resizes based on window width. It has four sizes based on UI Kit's internal breakpoints. For small and medium breakpoints its width is 100%, large its width is 900px, extra large and extra extra large 1024px, and above extra large 1440px. In the initial stage of developing the layout we'll use the [stitches `styled` function](https://stitches.dev/docs/api#styled) that ships with UI Kit to make the container easier to visualize.
```

Comment:
nit: For small and medium breakpoints the container's width is 100%, large is 900px, extra large and extra extra large is 1024px, and above extra large is 1440px.

Reply:
changed in b32ecd8